catalog/video/video-mpegts-stream/src/main/java/org/codice/alliance/video/stream/mpegts/netty/RawUdpDataToMTSPacketDecoder.java

```diff
@@ -94,18 +94,18 @@ public void channelActive(ChannelHandlerContext ctx) throws Exception {
                 .buffer(BUFFER_SIZE);
     }
 
-    private Subject getGuestSubject(String ipAddress) {
+    private Subject getSecuritySubject(String ipAddress) {
         Subject subject = null;
-        GuestAuthenticationToken token =
-                new GuestAuthenticationToken(BaseAuthenticationToken.DEFAULT_REALM, ipAddress);
-        LOGGER.debug("Getting new Guest user token for {}", ipAddress);
+        VideographerAuthenticationToken token =
+                new VideographerAuthenticationToken(BaseAuthenticationToken.DEFAULT_REALM, ipAddress);
+        LOGGER.debug("Getting new videographer user token for ip address {}: token={}", ipAddress, token);
         try {
             SecurityManager securityManager = getSecurityManager();
             if (securityManager != null) {
                 subject = securityManager.getSubject(token);
             }
         } catch (SecurityServiceException sse) {
-            LOGGER.warn("Unable to request subject for guest user.", sse);
+            LOGGER.warn("Unable to request subject for videographer user.", sse);
```

Comment:
Without a subject there is no point in continuing.  This condition should fail the stream.  This is probably a different ticket, but we should probably report this as a failure on the stream in the admin plugin.